### PR TITLE
fix: Update test_meshing_utilities

### PR DESCRIPTION
**Pull request by hpohekar**
Update`test_meshing_utilities` to avoid failure due to variation in output.

### CODE REVIEW

**Comment by hpohekar** [comment]
@prmukherj I have pulled all merged PRs and started nightly dev test workflow on this PR branch [here](https://github.com/ansys/pyfluent/actions/runs/10419134675).

**Comment by seanpearsonuk** [comment]
I had not noticed that we are embedding "fdl" in public names.

**Review by seanpearsonuk — APPROVED** [review]

**Review by prmukherj — APPROVED** [review]

File: tests/test_meshing_utilities.py [hunk 1]
```diff
@@ -11,7 +11,6 @@ def pytest_approx(expected):
 
 @pytest.mark.skip("Activate with: https://github.com/ansys/pyfluent/pull/3205")
```
Comment (prmukherj) [inline tests/test_meshing_utilities.py:12]: @hpohekar, this test is getting skipped now, right? We need to activate it? And should it run for all PRs or only for nightly?

Comment (hpohekar) [reply tests/test_meshing_utilities.py:12]: @prmukherj I think this is skipped after pulling latest changes from main branch, I will enable it again. We can run it for all PRs as it takes less than a minute.

Comment (hpohekar) [inline tests/test_meshing_utilities.py:12]:
```suggestion
```

**Pull request merged**